data/aerial-photography/doq/index.html

```diff
@@ -131,10 +129,10 @@ <h3 id="SCSPhotomaps">
     </h3>
   </div>
   <div class="grid__col grid__col--12-of-12 package-content">
-    <p>AGRC has 1-meter Black and White Soil Conservation Service (SCS) Photomaps from 1936-1952. This is not a statewide coverage. The Photomaps have a UTM NAD27 projection. The date the photo was taken can be found for some of the tiles in the <code>Year</code> field of the <a href="ftp://ftp.agrc.utah.gov/Imagery/UGS_SCS_Historical/UGS_SCS_Historical_Index.zip">tile-index</a>.</p>
+    <p>AGRC has 1-meter Black and White Soil Conservation Service (SCS) Photomaps from 1936-1952. This is not a statewide coverage. The Photomaps have a UTM NAD27 projection. The date the photo was taken can be found for some of the tiles in the <code>Year</code> field of the <a href="https://storage.googleapis.com/state-of-utah-ugs-historic-imagery/imagery/SCS/UGS_SCS_Historical_Index.zip">tile-index</a>.</p>
     <h5>Related Resources</h5>
     <ul class="dotless">
-      <li><a href="{{ "ftp://ftp.agrc.utah.gov/Imagery/UGS_SCS_Historical/UGS_SCS_Historical_Index.zip" }}">A Shapefile of the available imagery tile boundaries</a>
+      <li><a href="{{ "https://raster.utah.gov/?cat=Historical" }}">Download imagery tiles via interactive map</a>
```

Comment:
we don't need the `{{` unless there is liquid stuff
```suggestion
      <li><a href="https://raster.utah.gov/?cat=Historical">Download imagery tiles via interactive map</a>
```

Reply:
a lot of hacking on that page there was

Reply:
are you going to accept the suggestin?